src/frontend/src/components/AdminButtons/AdminButtons.jsx

```diff
@@ -0,0 +1,41 @@
+import React, { useContext } from 'react';
+import { makeStyles } from '@material-ui/core/styles';
+import { IconButton } from '@material-ui/core';
+import FlagIcon from '@material-ui/icons/Flag';
+import DeleteIcon from '@material-ui/icons/Delete';
+import { UserStateContext } from '../../contexts/User/UserContext';
+
+const useStyles = makeStyles(() => ({
+  iconDiv: {
+    float: 'right',
+  },
+  iconSpan: {
+    paddingLeft: '10px',
+    paddingRight: '10px',
+  },
+}));
+
+function AdminButtons() {
+  const classes = useStyles();
+  const user = useContext(UserStateContext);
+  if (!user.isAdmin) {
+    return null;
+  }
+
+  return (
```

Comment:
you can use [short circuit evaluation](https://dev.to/superhackerman/short-circuit-evaluation-with-react-3dn4) here and remove the if statement above.
```jsx
return (
  {user.isAdmin && ...
```